backend/python/app/tests/helpers/comment_helpers.py

```diff
@@ -0,0 +1,27 @@
+from collections.abc import Mapping
+
+from ...models.comment import Comment
+from ...models.comment_all import CommentAll
+
+
+class CreateCommentDTO:
+    def __init__(self, story_translation_content_id, content):
+        self.story_translation_content_id = story_translation_content_id
+        self.content = content
+
+
+class UpdateCommentRequestDTO(Mapping):
+    def __init__(self, comment):
+        self.id = comment.id
+        self.resolved = comment.resolved
+        self.content = comment.content
+        self._dict = dict(id=comment.id, content=comment.content)
+
+    def __getitem__(self, key):
```

Comment:
nit: copy the comment from StoryTranslationContentRequestDTO
`# These functions are necessary for bulk_update_mappings to work`